addons/website_slides/static/src/js/website_slides.editor.js

```diff
@@ -10,55 +10,46 @@ var _t = core._t;
 
 WebsiteNewMenu.include({
     actions: _.extend({}, WebsiteNewMenu.prototype.actions || {}, {
-        new_slide: '_createNewSlide',
+        new_slide_channel: '_createNewSlideChannel',
     }),
     xmlDependencies: WebsiteNewMenu.prototype.xmlDependencies.concat(
-        ['/website_slides/static/src/xml/website_slides.xml']
+        ['/website_slides/static/src/xml/website_slide_channel.xml']
```

Comment:
slide_channel_upload.xml ?

Reply:
it contains the template of the modal to create new channel. I would keep it like this.